spec/content/kibana/spec.yml

```diff
@@ -47,6 +47,16 @@ spec:
       type: file
       contentMediaType: "application/json"
       pattern: '^.+\.json$'
+  - description: Folder containing AI assistant knowledge base content in markdown format
+    type: folder
+    name: knowledge_base
+    required: false
+    contents:
+      - description: A markdown file containing AI assistant context
+        type: file
+        contentMediaType: "text/markdown"
+        pattern: '^.+\.md$'
+    additionalContents: false # Disallows nested directories
 versions:
```

Comment:
Thinking if we need to restrict this folder in existing versions of Kibana.
I guess existing versions of Kibana will ignore it, right? If including this folder in a package will break it in existing versions, we will need to add a version patch. If not, we are fine like this.

Could you please check if something goes wrong when installing a package with knowledge bases in a released version of Kibana?